frontend/src/features/properties/components/forms/ParcelDetailForm.tsx

```diff
@@ -201,10 +201,18 @@ const ParcelDetailForm = (props: ParcelPropertyProps) => {
       pidDuplicated = !(await isPidAvailable(values));
     }
 
+    let pinDuplicated = false;
```

Comment:
I missed this on Goeme's review but I think this is important enough that it shouldn't be ignored.

Since we are performing expensive, field specific validation I think we should be doing that validation at the level of the problematic field and not of the whole form:

https://formik.org/docs/guides/validation#field-level-validation

Reply:
That way we aren't performing this validation every time the form changes (or at least, checking to see if we need to perform this validation every time the form changes).

Reply:
Some more context on this since it is relevant and not obvious.

We rely heavily on memoized form components in most of our forms for performance reasons. The consequence of this is that a change in field A will not trigger a re-render in field B. This means that any component that needs to be triggered by another component needs to be handled at the global form level (ie evaluation sums).

In this case I think the validation logic is specific to each individual field, and so I don't think the above will be an issue for field based validation.

Reply:
Ah ok this makes sense, I'll try to switch it over to field level validation!

Reply:
Would it be bad practice to just add the following: 
` if (values.pin && initialValues.pin !== values.pin) {
      pinDuplicated = !(await isPinAvailable(values));
  }`? 
I noticed the PID was not firing every time to check for duplicate but mine was as I did not check it against its initial value